test/e2e/helpers.js

```diff
@@ -531,17 +531,7 @@ const PRIVATE_KEY_TWO =
 const ACCOUNT_1 = '0x5cfe73b6021e818b776b421b1c4db2474086a7e1';
 const ACCOUNT_2 = '0x09781764c08de8ca82e156bbf156a3ca217c7950';
 
-const defaultGanacheOptions = {
-  accounts: [
-    {
-      secretKey: PRIVATE_KEY,
-      balance: convertETHToHexGwei(DEFAULT_GANACHE_ETH_BALANCE_DEC),
-    },
-  ],
-};
-
 const defaultGanacheOptionsForType2Transactions = {
-  ...defaultGanacheOptions,
   // EVM version that supports type 2 transactions (EIP1559)
   hardfork: 'london',
```

Comment:
Nit: You might be able to remove this one as well right now. The default hardfork in ganache already supports type2 transactions.

Reply:
good observation !! So at the time the Ganache class was defined, it was set as `hardfork: 'muirGlacier',`  as default, so the majority of specs are using that, unless specifically passing the london flag. In Anvil I also set the same hardfork as default, as lots of specs were breaking otherwise, as were relying to type 1 (gas checks and conf screen). We could change that and use london as default in the class, as I guess most of the cases should use type 2 now.
I'll take  a note on that, if that make sense I can update it in a separate PR? I'll wait on your feedback, thank you in advance :pray:

Reply:
:information_source:  Added a task here for me to address this: https://github.com/MetaMask/MetaMask-planning/issues/4368